app/src/main/java/org/gdg/frisbee/android/gde/GdeActivity.java

```diff
@@ -114,7 +120,7 @@ public void run() {
                 mViewPager.setAdapter(mViewPagerAdapter);
                 mTabLayout.setupWithViewPager(mViewPager);
 
-                getAchievementActionHandler().handleLookingForExperts();
+                //getAchievementActionHandler().handleLookingForExperts();
```

Comment:
Ok, but this is a general problem then. We cannot update the app until we fix it everywhere. Commenting this out will make it worse. I've just opened an issue for this #595